coordinator/src/main/java/org/apache/uniffle/coordinator/strategy/storage/AbstractSelectStorageStrategy.java

```diff
@@ -76,6 +92,50 @@ public void readAndWriteHdfsStorage(FileSystem fs, Path testPath,
     }
   }
 
+  @Override
+  public void detectStorage() {
+    uris = Lists.newCopyOnWriteArrayList(remoteStoragePathRankValue.entrySet());
+    if (remoteStoragePathRankValue.size() > 1) {
+      CountDownLatch countDownLatch = new CountDownLatch(uris.size());
+      uris.parallelStream().forEach(uri -> {
+        if (uri.getKey().startsWith(ApplicationManager.getPathSchema().get(0))) {
+          Path remotePath = new Path(uri.getKey());
+          String rssTest = uri.getKey() + "/rssTest-" + getCoordinatorId()
+              + Thread.currentThread().getName();
+          Path testPath = new Path(rssTest);
+          RankValue rankValue = remoteStoragePathRankValue.get(uri.getKey());
+          rankValue.setHealthy(new AtomicBoolean(true));
+          long startWriteTime = System.currentTimeMillis();
+          try {
+            FileSystem fs = HadoopFilesystemProvider.getFilesystem(remotePath, hdfsConf);
+            for (int j = 0; j < readAndWriteTimes; j++) {
```

Comment:
In the original logic in `AppBalanceSelectStorageStrategy`,  readAndWriteTimes should be 1. I am not sure if we should keep the original logic.

Reply:
It's my negligence.

Reply:
Do you have any suggestions here? @jerqi @advancedxy

Reply:
Ok for me.